lib/std/zig/system.zig

```diff
@@ -1294,6 +1310,39 @@ fn elfInt(is_64: bool, need_bswap: bool, int_32: anytype, int_64: anytype) @Type
     }
 }
 
+fn detectAndroidApiLevel() !u32 {
```

Comment:
Returning errors would probably be good enough in this function; I don't think we need the logging.